index.html

```diff
@@ -2471,6 +2471,25 @@ <h1>Speakers</h1>
                                 <span>KDE</span>
                             </div>
                         </div>
+
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                                <div class="image-holder">
+                                    <img class="background-image" alt="Shubham Padia" src="img/shubhamp.jpg">
```

Comment:
@shubham-padia please use ShubhamPadia.jpg. It's the followed convention. :smile:
Please also provide preview link. Thanks

Reply:
done